.env

```diff
@@ -0,0 +1,5 @@
+DATABASE_URL=mongodb://localhost/scholl
```

Comment:
The `.env` file should be in gitignore. remove it from the code.

Reply:
done